### DISCUSSION TITLE
PrimeNg TreeTable Feature Requests: state storage

### DISCUSSION THREAD

**Discussion by scheelu** [primefaces/community#1472] [PrimeNG] [answered]
Hello, I'd like the Treetable to have a state storage system equivalent of the classic p-table, thanks !

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We'll wait for community upvotes to implement this feature.

**Reply by alexander793** [reply]
Hi,
How long are you going to wait to implement this feature? Or how many upvotes does it need?
